### DISCUSSION TITLE
Fundamentos do SQL server

### DISCUSSION THREAD

**Discussion by Felipe-stack-dev** [balta-io/forum#693] [Tira Dúvidas] [answered]
Bom dia tudo bom? Qual a versão do mysql usada no curso Fundamentos SQL server?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @Felipe-stack-dev, como vai?

No curso utilizamos o SQL Server (2022), mas qualquer versão acima da 18 é suportada!
Roda nativo no Windows ou via Docker no Windows, Mac e Linux!
